bpf/bpf_lxc.c

```diff
@@ -1436,22 +1437,11 @@ ipv6_policy(struct __ctx_buff *ctx, int ifindex, __u32 src_label,
 skip_policy_enforcement:
 #ifdef ENABLE_NODEPORT
 	if (ret == CT_NEW || ret == CT_REOPENED) {
-		bool dsr = false;
 # ifdef ENABLE_DSR
-		int ret2;
-
-		if (!revalidate_data(ctx, &data, &data_end, &ip6))
-			return DROP_INVALID;
-
-		ret2 = handle_dsr_v6(ctx, ip6, &dsr);
```

Comment:
Looks like this was the only call to `handle_dsr_v{4,6}`.

Reply:
Yup, and those functions have been renamed to `nodeport_extract_dsr_*()` now.